filezilla.sls

```diff
@@ -1,18 +1,22 @@
-# both 32-bit (x86) AND a 64-bit (AMD64) installer available
+# FileZilla has changed their download URLs
+# due to winrepo installer limitations you need to manually download the 86x + x64 installer from
+# https://filezilla-project.org/download.php?show_all=1
+# and put it on the winrepo on master to install it the 'salt://win/repo-ng/filezilla/...
+
+{% set version = '3.40.0' %}
 {% set PROGRAM_FILES = "%ProgramFiles%" %}
+
 filezilla:
-  {% for version in ['3.40.0','3.39.0','3.38.1'] %}
   '{{ version }}':
     full_name: 'FileZilla Client {{ version }}'
     {% if grains['cpuarch'] == 'AMD64' %}
-    installer: 'https://downloads.sourceforge.net/project/filezilla/FileZilla_Client/{{ version }}/FileZilla_{{ version }}_win64-setup.exe'
```

Comment:
The following urls worked for me:

https://download.filezilla-project.org/client/FileZilla_3.40.0_win64-setup.exe
https://download.filezilla-project.org/client/FileZilla_3.39.0_win64-setup.exe
https://download.filezilla-project.org/client/FileZilla_3.38.1_win64-setup.exe

So, maybe use this instead:
```
https://download.filezilla-project.org/client/FileZilla_{{ version }}_win64-setup.exe
```